pkg/provider/aws/resources/rds.go

```diff
@@ -186,6 +186,78 @@ func (subnetGroup *RdsSubnetGroup) Create(dag *core.ResourceGraph, params RdsSub
 	return nil
 }
 
+type RdsProxyCreateParams struct {
```

Comment:
Should this stuff live in rds_proxy.go for better separation (I know it's a bit of an unusual case)

Reply:
we could do that and also could split out packages in the future by service. For now im going to defer just for times sake, but i do agree we will need to figure out some sort of organizational packaging as we expand our services and what we support